src/main/webapp/app/text/text.component.html

```diff
@@ -0,0 +1,82 @@
+            </h2>
+        </div>
+        <div class="col-12 col-lg-4 text-right">
+            <div *ngIf="participation"><jhi-result [participation]="participation"></jhi-result></div><br>
+            <button class="btn btn-primary" (click)="saveText()" [hidden]="submission?.submitted" [disabled]="isSaving">
+                <ng-container *ngIf="!isSaving">
+                    <span class="fa fa-save"></span>
+                    <span jhiTranslate="entity.action.save">Save</span>
+                </ng-container>
+                <ng-container *ngIf="isSaving">
+                    <span class="fa fa-refresh"></span>
+                    <span>Saving...</span>
+                </ng-container>
+            </button>
+            <button class="btn btn-success" (click)="submit()"
+                    [disabled]="!submission"
+                    [hidden]="submission?.submitted || !isActive">Submit</button>
+            <button class="btn btn-warning" (click)="submit()"
+                    [disabled]="!submission"
+                    [hidden]="submission?.submitted || isActive">Submit (deadline missed)</button>
+        </div>
+    </div><br>
+
+    <div class="row" *ngIf="textExercise">
+        <div class="col">
+            <p class="mb-3"><span><strong>Problem Statement:</strong></span> {{ textExercise.problemStatement }}</p>
+        </div>
+        <div class="col text-right" *ngIf="!submission?.submitted"><p><strong>Max. Score: {{ textExercise.maxScore }}</strong></p></div>
+    </div>
+
+    <div class="row">
+        <div [ngClass]="{'col-9': submission?.submitted, 'col-12': !submission?.submitted}">
+            <textarea title="Solution" [(ngModel)]="answer" style="width: 100%; height: 50vh;" [readonly]="submission?.submitted" [disabled]="submission?.submitted"></textarea>
+            <!--<div class="assessments">-->
+                <!--<div class="assessments__container">-->
+                    <!--<div class="assessments__canvas">-->
+                        <!--<ng-container *ngIf="result?.rated && assessments && positions">-->
+                            <!--<div *ngFor="let assessment of assessments"-->
+                                 <!--[style.top.px]="positions[assessment.id].y"-->
+                                 <!--[style.left.px]="positions[assessment.id].x">-->
+                                <!--<i class="fa" [ngClass]="{'fa-check': assessment.credits > 0, 'fa-times': assessment.credits < 0}" *ngFor="let i of numberToArray(assessment.credits, 0)"></i>-->
+                                <!--<i class="fa half" [ngClass]="{'fa-check': assessment.credits > 0, 'fa-times': assessment.credits < 0}" *ngIf="assessment.credits % 1 !== 0"></i>-->
+                            <!--</div>-->
+                        <!--</ng-container>-->
+                    <!--</div>-->
+                <!--</div>-->
+            <!--</div>-->
```

Comment:
Are you planning to extend the feature and use this template code?

Reply:
It's a placeholder, so if you are already implementing it I can remove it